executor/show.go

```diff
@@ -623,6 +623,10 @@ func (e *ShowExec) fetchShowCreateTable() error {
 		}
 	}
 
+	if tb.Meta().ShardRowID {
+		buf.WriteString(" SHARD_ROW_ID=1")
```

Comment:
if we enable shard_row_id by `SHARD_ROW_ID=2`?

Reply:
It's enabled too.